main.go

```diff
@@ -38,14 +39,14 @@ func run() ExitCode {
 
 	if err := buildStep.InstallDependencies(dependenciesOpts); err != nil {
 		buildStep.logger.Println()
-		buildStep.logger.Errorf(formattedError(fmt.Errorf("Failed to install Step dependencies: %w", err)))
+		buildStep.logger.Errorf(errorutil.FormattedError(fmt.Errorf("Failed to install Step dependencies: %w", err)))
 		return Failure
 	}
 
 	runOpts := createRunOptions(config)
 	if err := buildStep.Run(runOpts); err != nil {
 		buildStep.logger.Println()
-		logger.Errorf(formattedError(fmt.Errorf("Failed to execute Step main logic: %w", err)))
+		logger.Errorf(errorutil.FormattedError(fmt.Errorf("Failed to execute Step main logic: %w", err)))
```

Comment:
Similarly to this: https://github.com/bitrise-steplib/steps-xcode-test/pull/219#discussion_r1044560133